shudder/queue.py

```diff
@@ -19,37 +19,38 @@
 """
 import json
 
-from boto.utils import get_instance_metadata
-import boto.sns as sns
-import boto.sqs as sqs
+import boto3
 
 from shudder.config import CONFIG
+import shudder.metadata as metadata
 
 
-INSTANCE_ID = get_instance_metadata()['instance-id']
+INSTANCE_ID = metadata.get_instance_id()
 QUEUE_NAME = "{prefix}-{id}".format(prefix=CONFIG['sqs_prefix'],
                                     id=INSTANCE_ID)
 
 
 def create_queue():
-    """Creates the SQS queue and returns the connection/queue"""
-    conn = sqs.connect_to_region(CONFIG['region'])
-    queue = conn.create_queue(QUEUE_NAME)
-    queue.set_timeout(60 * 60)  # one hour
+    """Creates the SQS queue and returns the queue url and metadata"""
+    conn = boto3.client('sqs', region_name=CONFIG['region'])
+    queue_metadata = conn.create_queue(QueueName=QUEUE_NAME, Attributes={'VisibilityTimeout':'3600'})
```

Comment:
For whatever reason this doesn't return a queue object, beats the crap out of me why. Below we then convert this metadata into the object.